### PR TITLE
Remove remaining autobind decorated components

### PR DESCRIPTION
**Pull request by jackkav**
- [x] fix plugin load

### CODE REVIEW

File: packages/insomnia/src/ui/components/check-for-updates-button.tsx [hunk 1]
```diff
@@ -1,69 +1,44 @@
...
-        onClick={this._handleCheckForUpdates}
-      >
-        {status || children}
-      </button>
-    );
-  }
-}
+  };
+  useEffect(() => {
+    electron.ipcRenderer.on('updater.check.status', listenerCheckStatus);
```
Comment (marckong) [inline packages/insomnia/src/ui/components/check-for-updates-button.tsx:25]: I am wondering if there was particular reason that we wanted to use the ipcRenderer here directly unlike how we did with websocket.

Comment (marckong) [reply packages/insomnia/src/ui/components/check-for-updates-button.tsx:25]: Oh that is not a scope of this PR.

File: packages/insomnia/src/ui/components/time-from-now.tsx [hunk 2]
```diff
@@ -12,56 +11,24 @@ interface Props {
...
-export class TimeFromNow extends PureComponent<Props, State> {
-  _interval: NodeJS.Timeout | null = null;
-
-  state: State = {
-    text: '',
-  };
-
-  _update() {
-    const { timestamp, titleCase } = this.props;
+export const TimeFromNow: FC<Props> = ({ className, timestamp, titleCase, intervalSeconds }) => {
```
Comment (marckong) [inline packages/insomnia/src/ui/components/time-from-now.tsx]: For some reasons, the TimeFromNow is rendering the time slowly as described in this video - https://www.loom.com/share/61fc791d759e4f7ab410fe0709d964bd

**Review by gatzjames — APPROVED** [review]
🚀

**Pull request merged**